src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Bau/BauCacheController.cs

```diff
@@ -0,0 +1,116 @@
+#nullable enable
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Common;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Cache;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Requests;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.IBlobStorageService;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Bau
+{
+    [Route("api/bau")]
+    [ApiController]
+    [Authorize]
+    public class BauCacheController : ControllerBase
+    {
+        private static readonly HashSet<string> AllowedPublicReleasePaths = new HashSet<string>
+        {
+            FileStoragePathUtils.PublicContentDataBlocksDirectory,
+            // TODO: EES-2865 Remove with other fast track code
+            FileStoragePathUtils.PublicContentFastTrackResultsDirectory,
+            FileStoragePathUtils.PublicContentSubjectMetaDirectory
+        };
+
+        private static readonly HashSet<string> AllowedPublicTreePaths = new HashSet<string>
+        {
+            PublicationTreeCacheKey.GetKey(PublicationTreeFilter.AnyData),
+            PublicationTreeCacheKey.GetKey(PublicationTreeFilter.LatestData),
+            PublicationTreeCacheKey.GetKey(null)
```

Comment:
This doesn't look right to me
`PublicationTreeCacheKey.GetKey(null)`

Reply:
Ignore me, I see it's an explict way of requesting the key for the other variation `publication-tree.json`

Reply:
To make this a little more palatable/clearer, it no longer passes in null. Instead, we now use a default arg from the key's constructor.